files/models.py

```
@@ -1739,6 +1742,18 @@ class Meta:
         verbose_name = "Index page featured"
         verbose_name_plural = "Index page featured"
 
+    def save(self, *args, **kwargs):
+        from users.validators import sanitize_html
+
+        if self.text:
+            self.text = sanitize_html(self.text)
+        super().save(*args, **kwargs)
+
+    class Meta:
```

Ruff (0.13.3)

Redefinition of unused `Meta` from line 1740

(F811)